portfolio/src/main/webapp/script.js

```diff
@@ -13,21 +13,27 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random travel destination and image of destination to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomDestination() {
+  const placesToVisit = 
+      ['Tokyo', 'London', 'Rome', 'Amsterdam', 'Taipei', 'Sichuan'];
 
-  const visit = 
-      ['Tokyo', 'London', 'Rome', 'New Amsterdam', 'Taipei', 'Sichuan'];
+  const visit_pics = {'Tokyo': 'https://cdn.pixabay.com/photo/2020/01/15/19/03/shinjuku-4768674_960_720.jpg',
```

Comment:
Please use `lowerCamelCase` for this per the [style guide](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names).

Or move it outside of the function and use `CONSTANT_CASE`